.github/workflows/prod.backup.yml

```diff
@@ -0,0 +1,29 @@
+name: PROD - Create database backup
+on:
+  schedule:
+    - cron: '0 7,19 * * *' # Twice a day: 7am and 7pm UTC
+  workflow_dispatch:
+jobs:
+  make_backup:
+    runs-on: ubuntu-18.04
+    container:
+      image: registry.gitlab.com/yldio/asap-hub/squidex-utils
+    env:
+      SQUIDEX_APP: 'asap-hub'
+      BUCKET_NAME: 'asap-hub-prod-data-backup'
+      SQUIDEX_CLIENT_ID: ${{ secrets.SQUIDEX_CLIENT_ID_PROD }}
+      SQUIDEX_CLIENT_SECRET: ${{ secrets.SQUIDEX_CLIENT_SECRET_PROD }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
```

Comment:
this is basically a copy of the dev workflow just with different params. You have probably looked into it but isn't there a way to make this reusable and not have to duplicate the code somehow? Unless only through an action? in which case we may want to consider make this job a private action

Reply:
I looked into it. I don't think the interface is clear enough. Plus, we want different cron frequencies for dev and prod.
Also I don't want to overcomplicate simple logic just because we want to keep it DRY

Reply:
i mean.. this is a huge amount of repetition so its not 'over complicating just because we want to keep it DRY'.

You've only just raised a PR for Docker changes and asked me to incorporate them because you didn't want to duplicate Docker config.

I would look into making this an action with params, as far as I am aware the workflow triggers and env vars can still be very much independent of the steps.

Reply:
Is this your attempt to avenge my complaint from your PR? 😄 

> You've only just raised a PR for Docker changes and asked me to incorporate them because you didn't want to duplicate Docker config.

This is incorrect. I suggested we could make the workflow simpler by leveraging docker. Use just one job instead of two. To which you even agreed.

Like I said, we want the backups to happen at different rates, which is not possible with just one action.

Reply:
I dont understand - actions are this https://docs.github.com/en/actions/creating-actions/about-actions - how is this related to workflow schedule rates? Unless im missing something here..

Reply:
We want the dev action to run once a week (https://github.com/yldio/asap-hub/pull/836/files#diff-6c5c55f22f119d90b63a7c93fa6ab557f0e95b1a8eac1b98c0ca8ccac95f9954R4) and the prod one to run twice a day (https://github.com/yldio/asap-hub/pull/836/files#diff-91c4c62ed059201e451aa08d9e9abfd2af8e1ad997cd25ebc940ac44a2a3402cR4). 

Would you have a way to have the same action trigger at two different rates for different environments?

Reply:
from the documentation
```
Actions are individual tasks that you can combine to create jobs and customize your workflow. You can create your own actions, or use and customize actions shared by the GitHub community.
```

This means that your `dev.backup.yml` and `prod.backup.yml` instead of having all of these steps duplicated in both files:
```yml
      - name: Add the configuration
        run: sq config add $SQUIDEX_APP $SQUIDEX_CLIENT_ID $SQUIDEX_CLIENT_SECRET
      - name: Run sync
        run: sq sync out ./backup
      - name: Tar the files
        run: |
          FILE_NAME=$(date +%s| cut -b1-13)-squidex-dev-backup.tar.gz && \
          tar -czf $FILE_NAME backup/ && \
          echo "FILE_NAME=$FILE_NAME" >> $GITHUB_ENV
      - name: upload to S3
        run: aws s3api put-object --private --bucket $BUCKET_NAME --key $FILE_NAME --body $FILE_NAME --storage-class STANDARD_IA
```
Would only have a step like this:
```yml
    - uses: actions/javascript-action@v1.0.1
 ```
As far as I am aware this has nothing to do with workflow triggers.